schemas/theia-extension.schema.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Theia package.json Schema",
+  "type": "object",
+  "$ref": "https://json.schemastore.org/package.json",
+  "required": [
+    "keywords",
+    "theiaExtensions"
+  ],
+  "properties": {
+    "keywords": {
+      "description": "This helps people discover your package as it's listed in 'npm search'. Must include 'theia-extension'.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
```

Comment:
Not sure if we need to re-specify these properties for `keywords` since it already exists in the base schema.